modules/guides/pages/read-data.adoc

```diff
@@ -0,0 +1,167 @@
+= Read Data
+:description: How to read documents in Couchbase.
+:page-pagination: full
+:page-topic-type: guide
+:tabs:
+:github: Click the GitHub button icon:github[] to view this code in context.
+
+[abstract]
+{description}
+
+Retrieving documents by ID is the fastest and simplest way to read data in Couchbase.
+The Data Service allows you to retrieve a full document when you need to fetch all of the data stored. However, in instances where this can be costly and unnecessary, Couchbase allows you to access only specific _paths_ within a document as well.
+
+== Full Documents
+
+To read a single document in Couchbase you can perform a `get` operation.
```

Comment:
Same description as `create-data.adoc`. Each procedure should start with "To do X", and each instruction in the procedure should be an imperative, "Do this!" — not "you can do this" or "we can do this".